app/assets/src/components/PipelineSampleReads.jsx

```diff
@@ -475,6 +491,27 @@ class PipelineSampleReads extends React.Component {
     let run_date_display_augmented = !date_available
       ? ""
       : "| data processed " + run_date_display;
+    let pipeline_versions =
```

Comment:
Would prefer this line as below. Key change is to avoid confusing React with other elements.

```
let pipeline_versions;
if (this.pipelineVersions.length > 1) {
  pipeline_versions = (
    <span>
      {" "}
      | switch to
      {this.pipelineVersions.map((version, i) => {
        const phash = { pipeline_version: version };
        if (version != this.props.reportPageParams.pipeline_version) {
          return (
            <a
              key={"version_" + i}
              onClick={() => {
                this.refreshPage(phash);
              }}
            >
              {" "}
              {version}{" "}
            </a>
          );
        }
      }, this)}
    </span>
  );
}
```